src/data_inclusion/schema/score_qualite.py

```diff
@@ -11,7 +11,7 @@
 )
 
 
-def adresse_bien_definie(service: Service) -> float | None:
+def adresse_bien_définie(service: Service) -> float | None:
```

Comment:
Pourquoi ? Le template ne suggère pas d'utiliser des accents dans le code

```suggestion
def adresse_bien_definie(service: Service) -> float | None:
```